lib/screens/profile_screens/security/security_screen.dart

```diff
@@ -82,12 +82,12 @@ class SecurityScreen extends StatelessWidget {
                           );
                         },
                       ),
-                      if (state.shouldShowExportRecoveryPhrase)
+                      if (true)
```

Comment:
this is used for local testing. 
@gguij004  make a release note to make sure we set this value back to state.shouldShowExportRecoveryPhrase before we release.